product_docs/docs/pgd/4/rel_notes/index.mdx

```diff
@@ -13,24 +13,24 @@ navigation:
 - pgd_4.0.1_rel_notes
 - pgd_4.0.0_rel_notes
 redirects:
-  - /pgd/latest/rel_notes/pgd_4.0.0_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.0.1_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.0.2_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.0.3_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.1.0_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.1.1_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.2.0_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.2.1_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.2.2_rel_notes/ 
-  - /pgd/latest/rel_notes/pgd_4.3.0_rel_notes/ 
+  - /pgd/latest/rel_notes/pgd_4.0.0_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.0.1_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.0.2_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.0.3_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.1.0_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.1.1_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.2.0_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.2.1_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.2.2_rel_notes/
+  - /pgd/latest/rel_notes/pgd_4.3.0_rel_notes/
 
 ---
 
 The EDB Postgres Distributed documentation describes the latest version of EDB Postgres Distributed 4, including minor releases and patches. The release notes provide information on what was new in each release. For new functionality introduced in a minor or patch release, the content also indicates the release that introduced the feature.
 
 | Release Date | EDB Postgres Distributed     | BDR   | HARP  | CLI | TPAexec                                                                          |
 | ------------ | ---------------------------- | ----- | ----- | --- | -------------------------------------------------------------------------------- |
-| 2023 Feb 14 | [4.3.0](pgd_4.3.0_rel_notes) | 4.3.0 | 2.2.1 | 1.1.0 | [23.9](https://techsupport.enterprisedb.com/customer_portal/sw/tpa/trunk/239/)   |
+| 2023 Feb 14 | [4.3.0](pgd_4.3.0_rel_notes) | 4.3.0 | 2.2.2[^*] | 1.1.0 | [23.9](https://techsupport.enterprisedb.com/customer_portal/sw/tpa/trunk/239/)   |
```

Comment:
Release date is still not clear for harp like when it was released confusing support and others. We need to have the separate row for harp release either by modifying this table or having separate table. I think Doc can help here. Otherwise for every release this will be problem. I think we have discussed similar thing for [v5 release](https://www.enterprisedb.com/docs/pgd/latest/rel_notes/) as well.

Reply:
Just to add, this tabular output is not only problem for harp but for cli, proxy, utilities etc...

Reply:
I'm not sure this is the best approach either... 
I mean if we had to release bdr 5.0.1 for the next release (mid-May) we would have PGD5.0.2 BDR5.0.1 HARP5.0.1 etc... 
I don't know what would be the best solution though :( 
Was this discussed with Petr or Kelly?

Reply:
> I mean if we had to release bdr 5.0.1 for the next release (mid-May) we would have PGD5.0.2 BDR5.0.1 HARP5.0.1

Yes, thats what we have mentioned [in this comment](https://github.com/EnterpriseDB/docs/pull/3796#discussion_r1138694435)

Reply:
Hi @drothery-edb @PJMODOS @smaher-edb @irionr , can we please conclude on this so that we can bring it to closure ?

Reply:
I kind of agree it's not ideal, however having combinations listed together likethis is at request of customers.